source/gui/settingsDialogs.py

```diff
@@ -1176,6 +1197,12 @@ def makeSettings(self, settingsSizer):
 	def postInit(self):
 		self.detectFormatAfterCursorCheckBox.SetFocus()
 
+	def onReportIndent(self, evt):
+		if self.lineIndentationCheckBox.GetValue():
+			self.indentTypeList.Enable(True)
+		else:
+			self.indentTypeList.Enable(False)

```

Comment:
Um, that overrides the method's name. Is this possible I.E. = rather 
than ( )


On 6/9/2016 4:40 AM, James Teh wrote:
>
> In source/gui/settingsDialogs.py 
> <https://github.com/nvaccess/nvda/pull/6057#discussion_r66418495>:
>
> > @@ -1176,6 +1197,12 @@ def makeSettings(self, settingsSizer):
> >  	def postInit(self):
> >  		self.detectFormatAfterCursorCheckBox.SetFocus()
> >
> > +	def onReportIndent(self, evt):
> > +		if self.lineIndentationCheckBox.GetValue():
> > +			self.indentTypeList.Enable(True)
> > +		else:
> > +			self.indentTypeList.Enable(False)
>
> nit: IMO, this is shorter/more readable:
>
> |self.indentTypeList.Enabled = self.lineIndentationCheckBox.IsChecked() |
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/6057/files/149618ac951d0380dad97bf4bd02ae1f00e451f9#r66418495>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivYJIFtF6gRUv8b5RzRJJzpq8RcNpks5qJ-2sgaJpZM4IxqMD>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194